actors/builtin/miner/miner_test.go

```diff
@@ -3617,6 +3908,41 @@ func (h *actorHarness) setProofType(proof abi.RegisteredSealProof) {
 	require.NoError(h.t, err)
 }
 
+func (h *actorHarness) setBeneficiary(rt *mock.Runtime, beneficiaryAddr addr.Address, beneficiary miner.BeneficiaryInfo) {
```

Comment:
I don't think you need this. Instead, update the harness benficiary address after successfully changing it through the method calls.